pkg/reconciler/trigger/trigger_test.go

```diff
@@ -17,8 +17,6 @@ limitations under the License.
 package trigger
 
 import (
-	//	"context"
```

Comment:
I'm confused why it shows this as being a change, since these were already removed in:
https://github.com/knative/eventing/pull/1097/files

Reply:
yes, it was a rebase